code/modules/spells/spell.dm

```diff
@@ -282,6 +285,11 @@ GLOBAL_LIST_INIT(spells, typesof(/obj/effect/proc_holder/spell)) //needed for th
 /obj/effect/proc_holder/spell/Initialize()
 	. = ..()
 
+	if(casting_clothes_override)
+		casting_clothes = casting_clothes_override
+	else
+		casting_clothes = casting_clothes_base
+
 	still_recharging_msg = "<span class='notice'>[name] is still recharging.</span>"
 	charge_counter = charge_max
```

Comment:
```suggestion
/obj/effect/proc_holder/spell/Initialize()
	. = ..()
	
	if(!casting_clothes_base)
		casting_clothes_base = typecacheof(list(/obj/item/clothing/suit/wizrobe,
			/obj/item/clothing/suit/space/hardsuit/wizard,
			/obj/item/clothing/head/wizard,
			/obj/item/clothing/head/helmet/space/hardsuit/wizard,
			/obj/item/clothing/suit/space/hardsuit/shielded/wizard,
			/obj/item/clothing/head/helmet/space/hardsuit/shielded/wizard))

	casting_clothes = casting_clothes_base

	still_recharging_msg = "<span class='notice'>[name] is still recharging.</span>"
	charge_counter = charge_max
```

Reply:
Couldnt you just use casting_clothes_base ||=